docs/CONFIG.md

```diff
@@ -1638,6 +1638,20 @@ env = "test" # Example
 ```
 env is an example user specified key-value pair
 
+## TxmAsService
+```toml
+[TxmAsService]
+Enabled = false # Default
+```
+
+
+### Enabled
+```toml
+Enabled = false # Default
+```
+Enabled turns Transaction Manager as a service feature on or off. When enabled exposes endpoint to submit arbitrary EVM transaction
```

Comment:
nit: same as a comment earlier. Also add context that when this is true, the node doesn't run as a regular core node, and only starts services that are needed for TXM to work.